.github/CODEOWNERS

```diff
@@ -336,7 +336,7 @@ receiver/splunkenterprisereceiver/                               @open-telemetry
 receiver/splunkhecreceiver/                                      @open-telemetry/collector-contrib-approvers @atoulme
 receiver/sqlqueryreceiver/                                       @open-telemetry/collector-contrib-approvers @dmitryax @crobert-1
 receiver/sqlserverreceiver/                                      @open-telemetry/collector-contrib-approvers @sincejune @crobert-1
-receiver/sshcheckreceiver/                                       @open-telemetry/collector-contrib-approvers
+receiver/sshcheckreceiver/                                       @open-telemetry/collector-contrib-approvers @ishaish103
```

Comment:
You shouldn't be updating this file by hand, it's generated - see the very top of the file.

Update metadata.yaml and run `make generate`

Reply:
Yes, I'm happy to be a sponsor

Reply:
I can be your second sponsor.